server/bleep/src/cache.rs

```diff
@@ -169,7 +169,7 @@ impl<'a> ChunkCache<'a> {
         let response = qdrant
             .scroll(&ScrollPoints {
                 collection_name: semantic::COLLECTION_NAME.to_string(),
-                limit: Some(1_000_000),
+                limit: Some(1_000),
```

Comment:
How does `limit` affect `Scroll` performance?

Reply:
It cuts off potential huge right tail here. In practice we don't really have more than 10s of chunks per file from my testing, so it's really just to keep things sensible.